pymeasure/adapters/prologix.py

```diff
@@ -89,6 +89,30 @@ class PrologixAdapter(VISAAdapter):
         sudo udevadm control --reload-rules
         sudo udevadm trigger
 
+    Since the Prologix adapter uses the same communication channel (an USB
+    CDC) for both, communication with the adapter itself, as well as
+    communication over GPIB, certain things need to be kept in mind:
+
+    - Operations that need to read from GPIB use the standard :read:
+      method.
+
+    - Operations that just read responses from the Prologix itself need to
+      add the parameter :prologix=True: to :read:; this avoids requesting
+      data from GPIB. This is also necessary when the adapter is put into
+      "listen-only" mode, where all GPIB traffic is automatically being
+      passed up.
+
+    - Binary data must be passed to the bus using :write_binary_values:.
+      This takes care of properly escaping those binary values that would
+      otherwise be interpreted by the Prologix adapter. Note that the
+      default for :write_binary_values: are to assume floating-point binary
+      data, and prepend IEEE headers. In order to pass just plain bytes to
+      the adapter, tune the :datatype: and :header_fmt: parameters:
```

Comment:
See above. The following will render as code blocks
~~~
:code:`something`
~~~